src/cases/store/actions/case-search.action.spec.ts

```diff
@@ -1,34 +1,41 @@
-import * as fromSearchCases from './case-search.action';
+
+import { TestBed } from '@angular/core/testing';
+import { BrowserDynamicTestingModule, platformBrowserDynamicTesting } from '@angular/platform-browser-dynamic/testing';
+import { JURISDICTION_SELECTED, JurisdictionSelected, RESET, Reset, APPLIED, Applied } from './case-search.action';
 
 describe('Case Search Actions', () => {
+  // beforeEach(() => {
```

Comment:
can we remove commented out unused code?

Reply:
Done, removed it. Thanks